others/RESOURCES.md

```diff
@@ -11,3 +11,13 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Documentaries about different technologies, Carrer  and Life advice by different developers, Interviews with creators of famous tools and technologies (Thomas Dohmke - GitHub CEO, Bjarne Stroustrup - Creator of C++)
+* [Fireship.io](https://www.youtube.com/@Fireship) - JavaScript, TypeScript, Code Reports, TypeScript, Code Report (Rubric about latest tech related news)
```

Comment:
Also I don't think this is a good description of Fireship's channel. His tutorial videos are on his second channel. Main one is predominantly for code reports and hyped technologies such as web frameworks and languages.

Reply:
I changed the description to a better one. I hope you like it this time.